backend/classes/production/sefilm/version.txt

```diff
@@ -0,0 +1,9 @@
++-------------------------------------------------------------------------
```

Comment:
This file should not be committed. I thought I had removed the folder from .gitignore...